x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_indices.tsx

```diff
@@ -177,116 +167,92 @@ export const EngineIndices: React.FC = () => {
-                { defaultMessage: 'Filter indices' }
-              ),
-              schema: true,
-            },
+          return {};
+        }}
+        search={{
+          box: {
+            incremental: true,
+            placeholder: i18n.translate(
+              'xpack.enterpriseSearch.content.engine.indices.searchPlaceholder',
+              { defaultMessage: 'Filter indices' }
+            ),
+            schema: true,
+          },
+        }}
+        pagination
+        sorting
+      />
+      {removeIndexConfirm !== null && (
+        <EuiConfirmModal
+          onCancel={() => setConfirmRemoveIndex(null)}
+          onConfirm={() => {
+            removeIndexFromEngine(removeIndexConfirm);
+            setConfirmRemoveIndex(null);
+            sendEnterpriseSearchTelemetry({
+              action: 'clicked',
+              metric: 'entSearchContent-engines-indices-removeIndexConfirm',
+            });
           }}
-          pagination
-          sorting
-        />
-        {removeIndexConfirm !== null && (
-          <EuiConfirmModal
-            onCancel={() => setConfirmRemoveIndex(null)}
-            onConfirm={() => {
-              removeIndexFromEngine(removeIndexConfirm);
-              setConfirmRemoveIndex(null);
-              sendEnterpriseSearchTelemetry({
-                action: 'clicked',
-                metric: 'entSearchContent-engines-indices-removeIndexConfirm',
-              });
-            }}
-            title={i18n.translate(
-              'xpack.enterpriseSearch.content.engine.indices.removeIndexConfirm.title',
-              { defaultMessage: 'Remove this index from the Search Application' }
-            )}
-            buttonColor="danger"
-            cancelButtonText={CANCEL_BUTTON_LABEL}
-            confirmButtonText={i18n.translate(
-              'xpack.enterpriseSearch.content.engine.indices.removeIndexConfirm.text',
-              {
-                defaultMessage: 'Yes, Remove This Index',
-              }
-            )}
-            defaultFocusedButton="confirm"
-            maxWidth
-          >
-            <EuiText>
-              <p>
-                {i18n.translate(
-                  'xpack.enterpriseSearch.content.engine.indices.removeIndexConfirm.description',
-                  {
-                    defaultMessage:
-                      "This won't delete the index. You may add it back to this search application at a later time.",
-                  }
-                )}
-              </p>
-            </EuiText>
-          </EuiConfirmModal>
-        )}
-        {addIndicesFlyoutOpen && <AddIndicesFlyout onClose={closeAddIndicesFlyout} />}
-      </>
-    </EnterpriseSearchEnginesPageTemplate>
+          title={i18n.translate(
+            'xpack.enterpriseSearch.content.engine.indices.removeIndexConfirm.title',
+            { defaultMessage: 'Remove this index from the Search Application' }
```

Comment:
```suggestion
            { defaultMessage: 'Remove this index from the search application' }
```

Should this be lower case for consistency here?